src/csharp/SDK/Transformation.cs

```diff
@@ -142,6 +142,129 @@ public void DepthImageToColorCamera(Image depth, Image transformed)
+        /// <returns>A custom image transformed in to the color camera perspective.</returns>
+        public (Image transformedDepth, Image transformedCustom) DepthImageToColorCameraCustom(Image depth, Image custom, TransformationInterpolationType interpolationType, uint invalidCustomValue)
+        {
+            if (depth == null)
+            {
+                throw new ArgumentNullException(nameof(depth));
+            }
+
+            if (custom == null)
+            {
+                throw new ArgumentNullException(nameof(custom));
+            }
+
+            Image transformedDepth = new Image(
+                ImageFormat.Depth16,
+                this.calibration.ColorCameraCalibration.ResolutionWidth,
+                this.calibration.ColorCameraCalibration.ResolutionHeight,
+                this.calibration.ColorCameraCalibration.ResolutionWidth * sizeof(short))
+            {
+                DeviceTimestamp = depth.DeviceTimestamp,
+            };
+
+            int bytesPerPixel;
+            switch (custom.Format)
+            {
+                case ImageFormat.Custom8:
+                    bytesPerPixel = sizeof(byte);
+                    break;
+                case ImageFormat.Custom16:
+                    bytesPerPixel = sizeof(short);
+                    break;
+                default:
+                    throw new NotSupportedException("Failed to support this format of custom image!");
+            }
+
+            Image transformedCustom = new Image(
+                custom.Format,
+                this.calibration.ColorCameraCalibration.ResolutionWidth,
+                this.calibration.ColorCameraCalibration.ResolutionHeight,
+                this.calibration.ColorCameraCalibration.ResolutionWidth * bytesPerPixel)
+            {
+                DeviceTimestamp = custom.DeviceTimestamp,
+            };
+
+            this.DepthImageToColorCameraCustom(depth, custom, transformedDepth, transformedCustom, interpolationType, invalidCustomValue);
+
+            return (transformedDepth, transformedCustom);
+        }
+
+        /// <summary>
+        /// Transforms an Image and an Custom Image from the depth camera perspective to the color camera perspective.
+        /// </summary>
+        /// <param name="depth">Depth image to transform.</param>
+        /// <param name="custom">Custom image to transform.</param>
+        /// <param name="transformedDepth">An transformed deth image to hold the output.</param>
+        /// <param name="transformedCustom">An transformed custom image to hold the output.</param>
+        /// <param name="interpolationType">Parameter that controls how pixels in custom image should be interpolated when transformed to color camera space.</param>
+        /// <param name="invalidCustomValue">Defines the custom image pixel value that should be written to transformedCustom image in case the corresponding.</param>
```

Comment:
>Defines the custom image pixel value that should be written to transformedCustom image in case the corresponding. [](start = 45, length = 113)

Defines the custom image pixel value that should be written to transformedCustom in case the corresponding depth pixel can not be transformed into the color camera space.

Reply:
Fixed! Thanks,